examples/demos/example32-summary-rows.js

```diff
@@ -0,0 +1,167 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import ReactDataGrid, { SelectColumn } from 'react-data-grid';
+import Wrapper from './Wrapper';
+
+function DecimalFormatter({ value, isSummaryRow, maximumFractionDigits }) {
+  const text = value.toLocaleString('en-US', maximumFractionDigits == null ? { maximumFractionDigits: 2 } : { maximumFractionDigits: 0 });
+  return isSummaryRow ? <strong>{text}</strong> : <div>{text}</div>;
```

Comment:
@MayhemYDG thoughts on the this API where we pass `isSummaryRow ` prop to `Row/Cell/Formatters`? Another option is to have a dedicated `column.GroupFormatter`

Reply:
I'm fine with it.